src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -1204,5 +1212,32 @@ private void PutConnectionInPool()
 
             _pool.PutConnection(this);
         }
+
+        private struct ValueStringBuilder
+        {
+            public char[] Chars;
+            public int Offset;
```

Comment:
Nit: `Length` to align with `StringBuilder`? `Offset` makes me think of "start index" like `ArraySegment<T>.Offset`.